### PR TITLE
change to PM4

### PR DESCRIPTION
**Pull request by Virvolta**
everything works except that it makes an invisible item if the pack is put in pmmp but if you download the pack before and join it works

### CODE REVIEW

**Comment by Virvolta** [comment]
if the pack is put in pmmp
image

if the pack is put on before joining **it's like this because of the size of the image**
image

**Comment by Virvolta** [comment]
It's just my Minecraft maybe, I had my colleague test it worked perfectly

File: src/alvin0319/CustomItemLoader/CustomItemLoader.php [hunk 1]
```diff
@@ -57,7 +57,8 @@ public function getResourcePackFolder() : string{
 
 	public function onPlayerJoin(PlayerJoinEvent $event) : void{
 		$player = $event->getPlayer();
-		$player->getNetworkSession()->sendDataPacket(CustomItemManager::getInstance()->getPacket());
+        $packet = CustomItemManager::getInstance()->getPacket();
```
Comment (alvin0319) [inline src/alvin0319/CustomItemLoader/CustomItemLoader.php:60]: What is the purpose of this?

File: src/alvin0319/CustomItemLoader/CustomItemManager.php [hunk 2]
```diff
@@ -117,7 +117,7 @@ public function registerItem($item) : void{
 
 			$this->itemTypeEntries[] = new ItemTypeEntry($item->getProperties()->getNamespace(), $runtimeId, true);
 
-			$this->packetEntries[] = new ItemComponentPacketEntry($item->getProperties()->getNamespace(), new CacheableNbt($item->getProperties()->getNbt()));
+			$this->packetEntries[] = new ItemComponentPacketEntry($item->getProperties()->getNamespace(), $item->getProperties()->getNbt());
```
Comment (alvin0319) [inline src/alvin0319/CustomItemLoader/CustomItemManager.php:120]: This is not correct. ItemComponentPacketEntry requires wrapped NBT by CachableNbt.

https://github.com/pmmp/BedrockProtocol/blob/e777b5f19946d878c6286a65877e748cf0c57079/src/types/ItemComponentPacketEntry.php#L32-L35

File: src/alvin0319/CustomItemLoader/CustomItemLoader.php (hunk 1 above)
Comment (Virvolta) [reply src/alvin0319/CustomItemLoader/CustomItemLoader.php:60]: i have test result with var_dump

File: src/alvin0319/CustomItemLoader/CustomItemManager.php (hunk 2 above)
Comment (Virvolta) [reply src/alvin0319/CustomItemLoader/CustomItemManager.php:120]: excuse me i have Pocketmine 4.0.0-BETA7

**Pull request closed without merging**